camera_calibration/package.xml

```diff
@@ -8,25 +10,22 @@
   <author>James Bowman</author>
   <author>Patrick Mihelich</author>
   <maintainer email="vincent.rabaud@gmail.com">Vincent Rabaud</maintainer>
-  <maintainer email="stevenmacenski@gmail.com">Steven Macenski</maintainer>
```

Comment:
Readd maintainers. We need these emails.

Reply:
Ah of course, I've got no idea why I would remove you?